src/platforms/java/common/configuration/index.mdx

```diff
@@ -348,3 +348,36 @@ sentry.proxy.port=8080 # default 80
 sentry.proxy.user=proxy-user
 sentry.proxy.pass=proxy-password
 ```
+
+## Configuring Offline Caching
+
+SDK can store events on the disk in case of network errors, and send them to Sentry on another SDK init if Sentry server is reachable.
```

Comment:
```suggestion
The SDK can store events on the disk in case of network errors, and send them to Sentry on another SDK init if the Sentry server is reachable.
```